clustering.Rmd

```diff
@@ -45,95 +49,147 @@ use the subgroups to generate new questions about the data and follow up with a
 courses.  
 
-> There are also so-called *semisupervised* tasks, \index{semisupervised} where only some of the data
-> come with labels/annotations, but the vast majority don't. The goal
-> is to try to uncover underlying structure in the data that allows one to
-> guess the missing labels. This sort of task is beneficial, for example, when one 
-> has an unlabelled data set that is too large to manually label, but one is willing to
-> provide a few informative example labels as a "seed" to guess the labels for all the data.
-
+As in the case of classification, 
+there are many possible methods that we could use to cluster our observations 
+to look for subgroups. 
+In this book, we will focus on the widely used K-means \index{K-means} algorithm.
+In your future studies, you might encounter hierarchical clustering,
+principle component analysis, multidimensional scaling, and more; 
+see the additional resources section at the end of the next chapter 
+for where to begin learning more about these other methods.
+
+> There are also so-called *semisupervised* tasks, \index{semisupervised} 
+> where only some of the data come with response variable labels/values, 
+> but the vast majority don't. 
+> The goal is to try to uncover underlying structure in the data 
+> that allows one to guess the missing labels. 
+> This sort of task is beneficial, for example, 
+> when onehas an unlabelled data set that is too large to manually label, 
+> but one is willing to provide a few informative example labels as a "seed" 
+> to guess the labels for all the data.
 
 **An illustrative example** 
 
-
 Here we will present an illustrative example using a data set \index{Palmer penguins} from the
 [{palmerpenguins} R data package](https://allisonhorst.github.io/palmerpenguins/). This data set was
 collected by [Dr. Kristen Gorman](https://www.uaf.edu/cfos/people/faculty/detail/kristen-gorman.php) and
-the [Palmer Station, Antarctica LTER](https://lternet.edu/) and includes
-measurements for adult penguins near Palmer Station [@palmerpenguins]. We have
+the [Palmer Station, Antarctica Long Term Ecological Research Site](https://pal.lternet.edu/) and includes
+measurements for adult penguins found near there [@palmerpenguins]. We have
 modified the data set for use in this chapter. Here we will focus on using two
 variables---penguin bill and flipper length, both in millimeters---to determine whether 
 there are distinct types of penguins in our data.
 Understanding this might help us with species discovery and classification in a data-driven
 way.
-```{r 09-penguins, echo = FALSE, message = FALSE, warning = FALSE, fig.height = 3, fig.width = 4, fig.cap = "Gentoo penguin. [@penguinsimage]", fig.retina = 2}
-knitr::include_graphics("img/gentoo.jpg")
+
+```{r 09-penguins, echo = FALSE, message = FALSE, warning = FALSE, fig.height = 3, fig.width = 4, fig.cap = "Gentoo penguin.", fig.retina = 2}
+knitr::include_graphics("img/Gentoo_Penguin._(8671680772).jpg")
```

Comment:
any chance we can keep the older / simpler filename of just `gentoo.jpg` ? parentheses in filenames is kind of annoying for common command line tasks

Reply:
Yes, I just left it as a record of where the image came from (that is what the image name was at the source. I will just put a comment above with the source link.